Snapper.NUnit/NUnitTestHelper.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using NUnit.Framework;
+
+namespace Snapper.NUnit
+{
+    public static class NUnitTestHelper

```

Comment:
Should this be public? I don't think class should be exposed through the nuget package

Reply:
I made it internal